frontend/src/js/ui-components/InputSelect/InputSelectComponents.tsx

```diff
@@ -37,14 +37,19 @@ export const ItemsInputContainer = styled("div")`
   width: 100%;
 `;
 
-export const Menu = styled("div")`
+export const MenuContainer = styled("div")`
   position: absolute;
+  width: 100%;
+  z-index: 3;
+  margin-top: 3px;
+  padding-bottom: 5px;
```

Comment:
I'd suggest to go for even some more space
```suggestion
  padding-bottom: 10px;
```